test/api/foreign_method_user_data.c

```diff
@@ -0,0 +1,36 @@
+#include <string.h>
+
+#include "foreign_method_user_data.h"
+
+static const double zero = 0;
+static const double answer = 42;
+static const double elite = 1337;
+static const double oneZeroOne = 101;
+static const double oneZeroThree = 103;
+static const double oneZeroFive = 105;
+
+static void test(WrenVM* vm, void *userData)
+{
+  wrenSetSlotDouble(vm, 0, *(double *)userData);
+}
+
+WrenBindForeignMethodResult foreignMethodUserDataBindMethod(const char* signature)
+{
+  WrenBindForeignMethodResult retval = {};
```

Comment:
wren_shared does not compile this. line should be = {0}

Reply:
[Is an empty initializer list valid C code?](https://stackoverflow.com/a/17589839/7884305)

Reply:
@Orcolom thanks!  Fixed.